.travis.yml

```diff
@@ -32,8 +32,6 @@ env:
     - ES6_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.2.2.tar.gz"
     - ES_HOST=127.0.0.1
   matrix:
-    # ES2 + SQLite
-    - REQUIREMENTS=lowest EXTRAS=all,sqlite,elasticsearch2 SQLALCHEMY_DATABASE_URI="sqlite:///test.db" ES_URL=$ES2_DOWNLOAD_URL
```

Comment:
I think you an also remove the `ES2_DOWNLOAD_URL` from above

Reply:
:+1: